source/_components/remember_the_milk.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "Remember The Milk"
+description: "Instructions on how to use Remember The Milk with Home Assistant."
+date: 2017-10-10 10:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.23
```

Comment:
At least `0.56`.

Reply:
OK, I was assuming this is updated automatically on merge...